src/Utilities/model.jl

```diff
@@ -303,10 +303,10 @@ function MOI.is_valid(model::AbstractModel, ci::MOI.ConstraintIndex)
 end
 
 function MOI.supports_constraint(
-    model::AbstractModel,
-    ::Type{F},
-    ::Type{S},
-) where {F<:MOI.AbstractFunction,S<:MOI.AbstractSet}
+    @nospecialize(model::AbstractModel),
+    @nospecialize(F::Type{<:MOI.AbstractFunction}),
```

Comment:
This one may be more dangerous

Reply:
Note that the inner method will be compiled. It's just this one that isn't.